tests/unit/jbi/whiteboard_actions/test_default.py

```diff
@@ -19,10 +22,34 @@ def test_default_returns_callable_without_data():
         assert callable_object()
 
 
-def test_default_returns_callable_with_data():
+def test_default_returns_callable_with_data(webhook_request_example, monkeypatch):
     callable_object = default.init(whiteboard_tag="", jira_project_key="")
     assert callable_object
+
+    def mock_getbug(self, id):
+        class MockBug:
+            __dict__ = webhook_request_example.bug
+
+        return MockBug()
+
+    def mock_update_bugs(self, id, updates):
+        return {}
+
+    def mock_create_issue(self, fields):
+        return {}
+
+    def mock_create_or_update_issue_remote_links(self, issue_key, link_url, title):
+        return {}
+
+    monkeypatch.setattr(bugzilla.Bugzilla, "getbug", mock_getbug)
+    monkeypatch.setattr(bugzilla.Bugzilla, "update_bugs", mock_update_bugs)
+    monkeypatch.setattr(Jira, "create_issue", mock_create_issue)
+    monkeypatch.setattr(
```

Comment:
I think that this is a bit weird. 

I have the feeling that you can do much simler with `unittest.mock` (*not tested*):

```py
from unittest import mock

def test_default_returns_callable_with_data(webhook_request_example):
    with mock.patch("src.jbi.whiteboard_actions.default.get_jira") as mocked_jira:
        with mock.patch("src.jbi.whiteboard_actions.default.get_bugzilla") as mocked_bz:
            mocked_bz.getbug.return_value = webhook_request_example.bug
            value = callable_object(payload=webhook_request_example)

    assert value["status"] == "..."   
```